### DISCUSSION TITLE
1:N view

### DISCUSSION THREAD

**Discussion by Michal-Mikolas** [AntlerVC/firetable#742] [Support / Q&A] [answered]
Hello, 
let's say I have a simple database with _Interprets_ and their _Songs_. One _Interpret_ can have multiple _Songs_ obviously. 

Now I want into my "_Interprets_ view" add to every row this button: `[Show this interpret's Songs]`. After clicking the button I will see rows from _Songs_ table, which belong to the selected _Interpret_. 

How can I do that? It's just simple 1:N relation. 

Thank you.

**Comment by harinij** [comment, marked as answer]
@Michal-Mikolas Hi 👋 To make sure I understand the question, when you click an "action" column button in the "Interprets" table view, you want to fetch all the rows from the Songs collection for that Intepret and add them to a subcollection of that Interpret row?

**Reply by Michal-Mikolas** [reply]
I just want to: 

1. Find interpret which I'm interested in (with filtering interprets by some criteria: genre, year of birth etc.). 
2. Via one-button click see all the songs of the chosen interpret.

**Reply by Michal-Mikolas** [reply]
So is it possible?

**Reply by harinij** [reply]
@Michal-Mikolas this is possible in multiple ways depending on how you want to use it after the songs are fetched.

**[Action column:](https://docs.rowy.io/field-types/action)** You can have an action button that does this, whenever a row's action button is click it fetches all the songs of that row's interpret and stored in an another json field.

**[Connector column:](https://docs.rowy.io/field-types/connector)** If you are looking for the list of songs to be shown as a drop-down menu option that the user can pick from, then it connector field would be useful
